pod-security/restricted/require-run-as-nonroot/require-run-as-nonroot.yaml

```diff
@@ -32,23 +32,23 @@ spec:
         anyPattern:
         - spec:
             securityContext:
-              runAsNonRoot: true
+              runAsNonRoot: "true"
```

Comment:
@vyankyGH why are these quoted strings here when they are type boolean in the k8s spec?

Reply:
This was for consistency. We may revisit later.